test/jsdom/misc.js

```diff
@@ -115,7 +115,7 @@ describe("jsdom/miscellaneous", () => {
       process.nextTick(() => {
         req.emit("response", res);
         process.nextTick(() => {
-          res.emit("data", "window.attachedHere = 123");
+          res.emit("data", new Buffer("window.attachedHere = 123"));
```

Comment:
Because the response is not expected to be already decoded as a string here. iconv-lite can only decode buffers so I change it to a buffer.